modules/strings/strings.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package strings
+
+import (
+	"strings"
+	"unicode/utf8"
+)
+
+// ToASCIIUpper returns s with all ASCII letters mapped to their upper case.
+func ToASCIIUpper(s string) string {
```

Comment:
Nothing in golang's stdlib for this?

Reply:
Unfortunately not, the closes things is ToUpperSpecial, but the struct that is passed is more or less magic to me and has a unexported field. This function is a modification from `ToUpper` with the same test cases.